cli/client.go

```diff
@@ -215,6 +215,34 @@ func (c *viamClient) getBillingConfig(cCtx *cli.Context, orgID string) error {
 	return nil
 }
 
+// OrganizationDisableBillingServiceAction corresponds to `organizations billing disable`.
+func OrganizationDisableBillingServiceAction(cCtx *cli.Context) error {
+	c, err := newViamClient(cCtx)
```

Comment:
[super nit] make variable more informative - like `client`

Reply:
It seems like this is used elsewhere in the code base so I guess it is ok for the sake of being consistent

Reply:
updated to be client